relay-profiling/src/error.rs

```diff
@@ -2,8 +2,8 @@ use thiserror::Error;
 
 #[derive(Debug, Error)]
 pub enum ProfileError {
-    #[error("invalid json in profile")]
-    InvalidJson(#[source] serde_json::Error),
+    #[error(transparent)]
+    InvalidJSON(#[from] serde_json::Error),
```

Comment:
We usually keep acronyms camel case, see 

https://github.com/getsentry/relay/blob/4fc80841346a088c40bb4cd69165e80b5e8862f7/relay-kafka/src/producer/mod.rs#L38

and 

https://rust-lang.github.io/api-guidelines/naming.html